tests/test_conversions.py

```diff
@@ -85,6 +85,12 @@ def test_em():
     inline_tests('em', '*')
 
 
+def test_header_with_space():
+    assert md('<h3>\n\nHello</h3>') == '### Hello\n\n'
+    assert md('<h4>\n\nHello</h4>') == '#### Hello\n\n'
+    assert md('<h5>\n\nHello</h5>') == '##### Hello\n\n'
```

Comment:
Should you add tests with spaces before the closing tag?
Just to be thorough.

Reply:
Added in new commit